### DISCUSSION TITLE
Global component styles

### DISCUSSION THREAD

**Discussion by KamranKhan-Dev** [Budibase/budibase#4038] [Ideas]
Would you consider the ability to style components in a central place so that the style is applied globally to the entire site?

For example, at the moment, if I want to style a form element a certain way, then I am limited to the inline styles.  If I want all the forms on the site to have the same look and feel, I have to go through the painful process of copy pasting those inline styles to every occurrence of that component on the site.

Generally, inline styles a pretty limited and don't allow you to do basic things such as selectors and cascading.

**Comment by mjashanks** [comment]
We are definately going to do this. I'm just not sure when.

Highly Likely this year. Maybe in the first half of this year. Not this Quarter. :)
